Formula/xcprojectlint.rb

```diff
@@ -0,0 +1,43 @@
+class Xcprojectlint < Formula
+  desc "Xcode project linter"
+  homepage "https://github.com/americanexpress/xcprojectlint"
+  url "https://github.com/americanexpress/xcprojectlint.git",
+      tag:      "0.0.6",
+      revision: "d9dad85847f5ee9b2143565a17d9066bb44b4b29"
+  license "Apache-2.0"
+  head "https://github.com/americanexpress/xcprojectlint.git"
+
+  depends_on xcode: ["12.0", :build]
+
+  def install
+    system "make", "install", "prefix=#{prefix}"
+  end
+
+  test do
+    # Bad Xcode project
+    (testpath/"Bad.xcodeproj/project.pbxproj").write <<~EOS
+      {
+        objects = {
+          B4A1B4A825CF28FA00DF4293 = {isa = PBXGroup; children = (); sourceTree = "";};
+          B4A1B4A925CF2DFF00DF4293 = {isa = PBXGroup; children = (B4A1B4A825CF28FA00DF4293); sourceTree = "";};
+          B4A1B47B25CF0DD400DF4293 = {isa = PBXProject; buildConfigurationList = 0; compatibilityVersion = ""; developmentRegion = 0; hasScannedForEncodings = 0; knownRegions = (); mainGroup = B4A1B4A925CF2DFF00DF4293; productRefGroup = B4A1B4A925CF2DFF00DF4293; projectDirPath = ""; projectRoot = ""; targets = ();};
```

Comment:
Does this line need to go on like this, or can we introduce linebreaks here, after the `;`s?

Reply:
I added line breaks.